schedule/yast/xfs/xfs@svirt-xen-hvm.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: xfs
+description: >
+  Installation with default parameters, except XFS being selected as filesystem
+  for the root partition. Installation is validated by successful boot and that
+  YaST does not report any issue.
+vars:
+  DESKTOP: gnome
+  FILESYSTEM: xfs
+schedule:
+  - installation/isosize
+  - installation/bootloader_svirt
```

Comment:
i just noticed that you could use bootloader_start here and in other places as well but it is fine to go on as it is

Reply:
As we tons of jobs, it's a good first step to move all scenarios to yaml, and then act on optimizations. I believe that we could also improve a lot on variables.